examples/03_connectivity/plot_seed_based_connectome.py

```diff
@@ -60,31 +92,84 @@
     adhd.func[0], confounds=adhd.confounds[0])
 
 ###############################################################################
-# Extract and plot correlation matrix
+# Correlation matrices estimation
+# -------------------------------
+# All starts with estimation of the signals **covariance**. Here the number of
+# samples
+print(power_timeseries.shape[0])
+
+###############################################################################
+# is less than the number of ROIs for Power atlas
+# (and of the same order for Dosenbach atlas). In these situations,
+# graphical lasso **sparse inverse covariance** estimator captures
+# well the covariance **structure**.
+from sklearn.covariance import GraphLassoCV
+
+###############################################################################
+# This estimator can be encompassed into the connectivities estimator object
+# :class:`nilearn.connectome.ConnectivityMeasure`. We set the connectivity
+# *kind* to correlation, to normalize covariance coefficients. 
+
+from nilearn import connectome
+
+correlation_estimator = connectome.ConnectivityMeasure(
+    cov_estimator=GraphLassoCV(verbose=True), kind='correlation')
```

Comment:
I agree: I had checked before, and the GraphLassoCV does look better.

You know what: if we stick to the principle that we'll use the covariance
estimators and not ConnectivityMeasure for single-subject analysis, than
we should use the GraphLassoCV and not ConnectivityMeasure. It is not
more complex to use the GraphLassoCV than to use LedoitWolf.

And there is a nice thing here, which is that for group studies, I would
recommend the LedoitWolf, as GraphLassoCV has instabilities that will
vary from subject to subject, however, for
single-subject, this clearly ilustrates that we should prefer
GraphLassoCV. So, if we teach people to use ConnectivityMeasure for group
analysis, and do things manually in the single subject case, it makes
sense.